### DISCUSSION TITLE
StickyBoard 사용 관련

### DISCUSSION THREAD

**Discussion by soaple** [innovationacademy-kr/42dashboard#6] [General]
이번 대시보드 구축에서 StickyBoard를 사용하는 것과 관련해서 카뎃분들의 의견을 받아보고 싶습니다.

먼저 이번에 기술 스택을 Nest.js와 TypeScript로 정했기 때문에,

StickyBoard와 기술 스택이 달라져서 오히려 개발에 장애물이 될 수도 있겠다는 생각이 들었습니다.

그래서 제 생각에는 StickyBoard 소스를 참고만 하고, 새로운 대시보드 프로젝트를 밑바닥부터 하나씩 만들어 나가면 어떨까 합니다.

여기에 댓글로 자유롭게 의견을 달아주시면 좋을 것 같습니다!

**Comment by Likilee** [comment]
말씀해주신 의견에 동의합니다.
다만 StickyBoard 프로젝트의 코드 구조, 데이터 흐름에 대해서 코드 리뷰형식으로 멘토님께서 한번 진행해주시면, 설계에 있어서 도움이 많이 될거 같은데 어떻게 생각하시나요? ㅇㅅㅇ

**Reply by soaple** [reply]
네, 그 부분은 저도 필요하다고 생각합니다.

한 번 날을 잡고 설계와 관련해서 얘기해보는 시간을 가지면 좋을 것 같아요~

StickyBoard의 구조나 대시보드를 개발할 때 고려할 점들을 제가 먼저 설명해드리고,

카뎃분들께서 구상한 내용들을 함께 리뷰해보는 시간이 될 것 같습니다.

그리고 이참에 고정적으로 요일을 하나 정해서 매주 2시간 정도 제가 오프라인에 함께 참여하는 것도 좋을 것 같은데 어떠신가요?

**Reply by jungmyeong96** [reply]
매주 월요일에 스프린트 회의를 계획하고 있는데, 그 때 오시면 같이 코드적인 부분이나 방향성에 대해서 많은 얘기를 나눌 수 있을 것 같습니다!

멘토링 시간은 월요일 1시가 좋을 것 같고, 회의장소는 "1층 경복궁" 으로 예약해뒀습니다.

현재 장소 협의 중인데, 다음주는 변동이 없을 것 같아서 다음주 개포 클러스터 1시에 뵙도록 할게요!

**Reply by soaple** [reply]
네, 그럼 매주 월요일 1시에 함께 참여하도록 하겠습니다!

**Comment by hunjin-choi** [comment]
멘토님 감사합니다! 좋은 저녁 보내세요!
